<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAutoLogin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/icon.png</normaloff>:/image/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;ID</string>
       </property>
       <property name="buddy">
        <cstring>idEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="idEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Your ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Password</string>
       </property>
       <property name="buddy">
        <cstring>passEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Your password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Auth server</string>
       </property>
       <property name="buddy">
        <cstring>serverNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="serverNameEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>https://auth.server.example/</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="repeatCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Auto &amp;relogin time interval (min)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="repatSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2048</number>
       </property>
       <property name="value">
        <number>120</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="randomizeCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Rand&amp;omize! (min)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="randomizeSpinBox">
       <property name="maximum">
        <number>2048</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>Next login to %v[min]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="stopTimerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Stop timer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="logoutButton">
       <property name="text">
        <string>Log&amp;out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>&amp;Login</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Message</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="messageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="ipEdit">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renewIPButton">
       <property name="text">
        <string>Re&amp;new IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkIpButton">
       <property name="text">
        <string>Check &amp;IP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="autoToTrayCheckBox">
       <property name="text">
        <string>&amp;Minimize to systemtray when application starts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autoStartCheckBox">
       <property name="text">
        <string>Login with saved id and password when application &amp;starts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="saveIdCheckBox">
       <property name="text">
        <string>Enable save/load &amp;user id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="savePassCheckBox">
       <property name="text">
        <string>Enable save/load &amp;password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autoLogoutCheckBox">
       <property name="text">
        <string>Logout &amp;when application closes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="renewIPCheckBox">
       <property name="text">
        <string>Renew IP a&amp;fter Login/Logout ( Windows only )</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="saveConfigButton">
       <property name="text">
        <string>Save &amp;Config</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>&amp;About Qt</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="minimizePushButton">
       <property name="text">
        <string>Minimize to &amp;tray</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>serverNameEdit</tabstop>
  <tabstop>idEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>repeatCheckBox</tabstop>
  <tabstop>repatSpinBox</tabstop>
  <tabstop>randomizeCheckBox</tabstop>
  <tabstop>randomizeSpinBox</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>logoutButton</tabstop>
  <tabstop>stopTimerButton</tabstop>
  <tabstop>renewIPButton</tabstop>
  <tabstop>checkIpButton</tabstop>
  <tabstop>autoToTrayCheckBox</tabstop>
  <tabstop>autoStartCheckBox</tabstop>
  <tabstop>saveIdCheckBox</tabstop>
  <tabstop>savePassCheckBox</tabstop>
  <tabstop>autoLogoutCheckBox</tabstop>
  <tabstop>renewIPCheckBox</tabstop>
  <tabstop>saveConfigButton</tabstop>
  <tabstop>aboutButton</tabstop>
  <tabstop>minimizePushButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>ipEdit</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>serverNameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
